File: app/src/main/java/com/example/ogan/codedenim/gson/Module.java

package com.example.ogan.codedenim.gson;

import java.util.List;

public class Module {

    private int ModuleId;
    private int CourseId;
    private String ModuleName;
    private String ModuleDescription;
    private int ExpectedTime;
    private Course Course;
    private List<Topic> Topics;

    public int getModuleId() {
        return ModuleId;
    }

    public int getCourseId() {
        return CourseId;
    }

    public String getModuleName() {
        return ModuleName;
    }

    public String getModuleDescription() {
        return ModuleDescription;
    }

    public int getExpectedTime() {
        return ExpectedTime;
    }

    public Course getCourse() {
        return Course;
    }

    public List<Topic> getTopics() {
        return Topics;
    }
}

File: app/src/main/java/com/example/ogan/codedenim/gson/Module.kt
package com.example.ogan.codedenim.gson

data class Module (

    val ModuleId: Int?,
    val CourseId: Int?,
    val ModuleName: String?,
    val ModuleDescription: String?,
    val ExpectedTime: Int?,
    val Course: Course?,
    val Topics: List<Topic>?
)
